src/Core/Domain/Manufacturer/ValueObject/ManufacturerId.php

```diff
@@ -66,8 +66,11 @@ public function getValue()
      */
     private function assertIsIntegerGreaterThanZero($value)
     {
-        if (!is_int($value) || 0 >= $value) {
-            throw new ManufacturerConstraintException(sprintf('Invalid manufacturer id "%s".', var_export($value, true)));
+        if (!is_int($value) || 0 > $value) {
```

Comment:
This one I'm not sure of, I allowed the 0 value so that we can un-assign the manufacturer from the Product
However `ManufacturerId` is also used in `ManufacturerController` and related commands (although there are no tests nor check for this 0 value elsewhere)

So either we accept this modification, or I though of an alternative to create a class that would extend `ManufacturerId` and would be dedicated to store this 0 value, we could name it `EmptyManufacturerId` for instance and it would be used by `UpdateProductOptionsCommand::setManufacturerId` when 0 value is used

Reply:
@matks I would like your opinion on this one ^^